docs-client/src/containers/MethodPage/QueryEditor/index.css

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*
+  Forked from GraphiQL 1.4.2
+  https://github.com/graphql/graphiql/blob/cd6854352ac6beff57af76db7de38e8157ff13aa/packages/graphiql/src/css/app.css#L476...L497
```

Comment:
- Could you add the changes you've made?
- Could you add the copyright header of GraphiQL right after our copyright header?
  ```
  /*
   *  Copyright (c) 2021 GraphQL Contributors.
   *
   *  This source code is licensed under the MIT license found in the
   *  LICENSE file in the root directory of this source tree.
   */
  ```
- Could you add an entry in `/NOTICE.txt` (the 'Modified redistributions' section)?

Reply:
| Could you add the changes you've made?
- Are you talking about added changes?

Reply:
I meant the list of the changes you made after forking the `app.css`. :slightly_smiling_face:

Reply:
I'm not sure if I changed it as intended, but I separated the css (`lint.css`, `app.css` etc...) I got from graphiql in `index.css`.